### DISCUSSION TITLE
Which Kindle Serial Number(s) to Load into DeDRM Plugin

### DISCUSSION THREAD

**Discussion by COHikerGirl** [noDRM/DeDRM_tools#162] [Q&A]
Apologies if this posts twice -- it seems I lost the first attempt.

My elderly Dad passed away at 94 during the pandemic.
I am trying to recover and import his Kindle eBooks into my own Kindle account/library.
I have full access to his account and his Kindle, as well as my own.

When following the DeDRM instruction (in DeDRM_plugin_ReadMe.txt), and AFTER having loaded the DeDRM plugin from file, the next instruction is to, "For Kindle ebooks from an E-Ink based Kindle (e.g. Voyage), or books downloaded from the Amazon web site 'for transfer via USB' to an E-Ink base Kindle, you must enter the Kindle's serial number in the customisation dialog."

At that point, should I load BOTH my Dad's Kindle serial number AND my own Kindle serial number -- or just one or the other?  Since I may continue using DeDRM to save off accessible versions of my own Kindle library, after this initial work to transfer books from my Dad's Kindle library into my own, it SEEMS the presumptive answer would be, "Both."  Am I correct?

**Comment by ElleKayEm** [comment]
Yes, I'd do both.  Books downloaded from his account and choosing his E-ink Kindle for the device will decrypt with the serial number for his Kindle.  And books downloaded from your account and choosing your Kindle will decrypt with your Kindle's serial number.
